lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -125,6 +125,8 @@ def __init__(
         self._pseudo_w, self._pseudo_h = pseudo_bbox[1] - pseudo_bbox[0]
 
         self.figure.add_axes(self)
+        for k in self.spines.keys():
+            self.spines[k].set_visible(False)
```

Comment:
Could you add a comment here saying something along the lines of "mplot3d currently manages its own spines and needs these turned off for bounding box calculations"?